Header header
time start
int32 waypointId
float64 waypointX
float64 waypointY
float64 waypointTheta
string info
